languages/cs.ini

```diff
@@ -63,6 +64,7 @@ All Pages Loaded = "Všechny stránky nahrány"
 All Text = "Vše"
 alphabrowse_matches = "Výsledky"
 alphabrowselink_html = "Procházet záznamy podle %%index%% od <a href="%%url%%">%%from%%</a>."
+Always ask me = "Pokaždé se zeptat"
```

Comment:
"Zeptat se pokaždé"

Reply:
It is in profile when you are selecting your preferred library. In that context I think my translation is slightly better.  Because accented word should be "pokaždé" not "zeptat",